plugins/arDominionB5Plugin/js/fullWidthTreeView.js

```diff
@@ -334,6 +334,22 @@
 
         // Update the url, TODO save the state
         window.history.pushState(null, null, url);
+
+        // Get new document title for G4 Tracking
+        var doctitle = $("#main-column h1").first();
+
+        // Create a temporary element to parse the HTML
+        var tempElement = document.createElement("div");
+        tempElement.innerHTML = doctitle[0].outerHTML;
+
+        // Extract the text content without HTML tags and set title
+        document.title = tempElement.textContent;
```

Comment:
Same as the BS2 version, we shouldn't need the extra div element here and should be able to get the title without it.